Code/Engine/Core/Input/Implementation/InputManager.cpp

```diff
@@ -208,6 +209,7 @@ void ezInputManager::ResetInputSlotValues()
   // this is crucial for accumulating the new values and for resetting the input state later
   for (ezInputSlotsMap::Iterator it = GetInternals().s_InputSlots.GetIterator(); it.IsValid(); it.Next())
   {
+    it.Value().m_fValueOld = it.Value().m_fValue;
```

Comment:
can't think of a better solution atm, at least not an easy one

Reply:
Ok let's leave it like that for now.